File: backends/php-laravel/app/Domain/Notification/ValueObjects/NotificationSaveVo.php
<?php

/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */

declare(strict_types=1);

namespace App\Domain\Notification\ValueObjects;

use App\Support\Traits\Getter;
use App\Support\ValueObjects\AbstractValueObject;

/**
 * 通知更新ValueObjectクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\Domain\Notification\ValueObjects
 */
class NotificationSaveVo extends AbstractValueObject
{
    use Getter;

    private bool $ok = false;
    private string $message = 'SUCCESS';
    private int $id = 0;
    private int $updated = 0;
}

File: backends/java-spring/src/main/java/com/authorization/domain/notification/valueobjects/NotificationSaveVo.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.domain.notification.valueobjects;

import com.authorization.support.valueobjects.AbstractValueObject;
import lombok.Data;
import lombok.EqualsAndHashCode;

/**
 * 通知更新ValueObjectクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class NotificationSaveVo extends AbstractValueObject {

    private boolean ok;
    private String message = "SUCCESS";
    private long id;
    private int updated;
}
